nion/swift/model/Graphics.py

```diff
@@ -716,6 +717,15 @@ def stroke_color(self) -> typing.Optional[str]:
     @stroke_color.setter
     def stroke_color(self, value: typing.Optional[str]) -> None:
         self._set_persistent_property_value("stroke_color", value)
+        self.notify_property_changed("used_stroke_style")
+
+    @property
+    def stroke_width(self) -> typing.Optional[float]:
+        return typing.cast(typing.Optional[float], self._get_persistent_property_value("stroke_width"))
+
+    @stroke_width.setter
+    def stroke_width(self, value: typing.Optional[float]) -> None:
+        self._set_persistent_property_value("stroke_width", value)
```

Comment:
For symmetry of code, here should probably be a `used_stroke_width` property, too. `used_stroke_width` would be a computed value and would always be `float`, never `None`.

Then all of the places where you do `self.stroke_width or 1` can be replaced with `self.used_stroke_style`.

Along with this, you can probably change the return values of `used_stroke_style` and `used_fill_style` to be `str` instead of `str | None`.